driver/ixsystems/iscsi.py

```diff
@@ -178,7 +180,8 @@ def check_connection(self):
             ctx.__setattr__("project_only", "True")
             vols = cinderapi.volume_get_all(ctx)
             attached_truenas_vol_count = len([vol for vol in vols
-                                              if vol.host.find("@ixsystems-iscsi#") > 0 and vol.attach_status == 'attached'])
+                                              if vol.host and vol.host.find("@ixsystems-iscsi#") > 0
+                                              and vol.attach_status == 'attached'])
```

Comment:
Add default value for TrueNAS scale and fix some situation volume.host is none cause panic exception.